### PR TITLE
[Rolling out Centipede] Project 11 - 20

### PR DESCRIPTION
**Pull request by Alan32Liu**
Continuing #8801.

The third round of rolling out `Centipede`, which contains the following 10 projects (some are not from within Google):

1. `bloaty`,
2. `double-conversion`,
3. `zopfli`,
4. `guetzli`,
5. `jsonnet`,
6. `astc-encoder`,
7. `piex`,
8. `wuffs`,
9. `arrow`,
10. `myanmar-tools`.

Projects are selected because they are `C++` projects.

### CODE REVIEW

**Review by jonathanmetzman — APPROVED** [review]
LGTM

I'd say wait until the first two batches aree merged for a day or two before merging this

**Comment by Alan32Liu** [comment]
> LGTM
> 
> I'd say wait until the first two batches aree merged for a day or two before merging this

Sure! I plan to merge the 2nd one (#8801) on Monday and if it goes well, I will merge this on Wednesday.

**Pull request merged**
